app/views/items/_item.html.erb

```diff
@@ -1,4 +1,13 @@
 <div class=<%= "item-#{item.id}" %>>
-  <%= link_to item.name, user_item_path(item.user, item)%>
-  <%= link_to "",[item.user, item], method: :delete,remote: true, class: "glyphicon glyphicon-ok" %><br>
+  <div class="row">
+    <div class="col-md-2"></div>
+    <div class="col-md-2"></div>
```

Comment:
You can add an [offset](http://getbootstrap.com/css/#grid-offsetting) instead of blank `div` tags.